components/segment-bridge/base/kustomization.yaml

```diff
@@ -1,4 +1,5 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
+- rbac
```

Comment:
should not this be `./rbac` ?

Reply:
It's okay either way, but I have changed it to `./rbac` in segment-bridge directory.